privval/socket_listeners_test.go

```diff
@@ -97,19 +97,18 @@ func TestListenerTimeoutAccept(t *testing.T) {
 
 func TestListenerTimeoutReadWrite(t *testing.T) {
 	const (
-		// This needs to be long enough s.t. the Accept will definitely succeed:
```

Comment:
I find these comments helpful. Would you mind putting them back?

Reply:
I don't mind at all :)

Reply:
Maybe you forgot to push the changes?

Reply:
I was hesitant to touch the repo at all:

https://github.com/cometbft/cometbft/discussions/1424